src/lints/trait_method_unsafe_removed.ron

```diff
@@ -17,6 +17,12 @@ SemverQuery(
                             public_api @filter(op: "=", value: ["$true"])
                         }
 
+                        supertrait @fold @transform(op: "count") @filter(op: "=", value: ["$zero"]) {
+                            trait {
+                                importable_path @fold @transform(op: "count") @filter(op: "=", value: ["$zero"])
+                            }
+                        }
```

Comment:
One thing I just realized: I'm not 100% sure how this plays with supertraits that are from a foreign crate including `std` or `core`.

Could you throw in a test case where the trait under test has a std or core trait as a supertrait? Perhaps a supertrait like `std::convert::AsRef` could do the trick.

Reply:
Are you talking about a sealed trait with a `std` supertrait, or an unsealed trait with a `std` supertrait (or both)?

Reply:
Because we currently don't do cross-crate analysis, some `supertrait` or `trait` edges might be missing if the supertrait is foreign. If the edges have lower-than-expected cardinality like that, it might spuriously satisfy the `count = 0` condition and cause false-positives.

So I think the case to look at is something like "sealed trait with `std` supertrait." But given how long I needed to figure out the exact case here, probably both the sealed and unsealed cases are worth having in the test suite.